recipes/fontconfig/all/conanfile.py

```diff
@@ -26,8 +27,6 @@ def requirements(self):
             self.requires("libuuid/1.0.3")
 
     def configure(self):
-        if self.settings.os == "Windows":
-            raise ConanInvalidConfiguration("Windows builds are not supported.")
```

Comment:
There was certainly a reason why Windows builds were not possible. Got something fixed in the new version?

Reply:
Not sure, building works with both .91 and .92 on MinGW, but the .91 version fails to find any fonts on Windows because the path to the system font directory is not correctly applied because of said bug.

Reply:
Just read your issue. Seems to be the case.

However, you need to make this conditional on the version build, `2.13.91` does still not support Windows